guide/private-key-management/single-user-schemes.md

```diff
@@ -0,0 +1,326 @@
+   height = 600
+   layout = "full-width"
+%}
+
+# Personal schemes
+
+Now let's dive in and look at the various schemes that might be suitable for a personal wallet-application with a single-user to understand the differences at a high level.
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/schemes/personal-schemes.jpg"
+   retina = "/assets/images/guide/private-key-management/schemes/personal-schemes@2x.jpg"
+   alt-text = ""
+   caption = "Cloud backup, manual backup , signing-device, threshold-signatures and multi-key."
+   width = 2012
+   height = 400
+%}
+
+- [Automatic cloud backup]({{ '/guide/private-key-management/single-user-schemes/#automatic-cloud-backup' | relative_url }})
+- [Manual backup / Recovery phrase]({{ '/guide/private-key-management/single-user-schemes/#recovery-phrase' | relative_url }})
+- [External signing device]({{ '/guide/private-key-management/single-user-schemes/#external-signing-device' | relative_url }})
+- [Threshold signatures / Key-sharing]({{ '/guide/private-key-management/single-user-schemes/#threshold-signatures--key-sharing' | relative_url }})
+- [Multi-key]({{ '/guide/private-key-management/single-user-schemes/#multi-key' | relative_url }})
+
+***
+
+## Automatic cloud backup
+
+By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new wallet. This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or recovery-phrase that is used to initiate a wallet. 
+
+The target audience with this scheme would be new or novice bitcoin users to whom being exposed to backing up a recovery-phrase as part of onboarding can be baffling and backfire. It could also be a starting point for a progressive security model that lets customers upgrade when the start accumulating more funds. 
+
+The automatic cloud backup scheme gives users reasonable security against theft, while providing low risk for self-inflicted losses. 
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/schemes/cloud-backup.jpg"
+   retina = "/assets/images/guide/private-key-management/schemes/cloud-backup@2x.jpg"
+   alt-text = "Automatic cloud backup illustration"
+   caption = "Private keys are encrypted and stored in the cloud."
+   width = 800
+   height = 400
+%}
+
+#### Pros 
+- Low onboarding friction
+- Low risk of self-inflicted loss due to the automatic nature and redundant backups
+
+#### Cons
+- Private key in cloud storage increases risk of malicious third party access
+- User needs to trust their OS and cloud storage provider
+- Unless the backup includes the recovery phrase users will not be able to export their wallet to another product
+
+### How it works
+The wallet-application will generate a wallet with a keypair, then encrypt and back the key up to a location that the user is unlikely to lose access to, while hard for a malicious third party to gain access to. 
```

Comment:
Since this is significant paragraph, I propose it's broken down into simple  to understand sentences that explain process. Suggestion below, maybe that's now what you're looking for.

```suggestion
First the application generates a wallet with a keypair. Then, the key is encrypted and backed up to a location convenient and safe for the user, yet hard to for a malicious third-party to gain access to.

```